hathor/verification/verification_service.py

```diff
@@ -12,15 +12,40 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-from hathor.transaction import BaseTransaction, Block, Transaction, TxVersion
+from typing import NamedTuple
+
+from hathor.conf.settings import HathorSettings
+from hathor.transaction import BaseTransaction, Block, MergeMinedBlock, Transaction, TxVersion
 from hathor.transaction.exceptions import TxValidationError
 from hathor.transaction.token_creation_tx import TokenCreationTransaction
 from hathor.transaction.validation_state import ValidationState
-from hathor.verification import block_verification, token_creation_transaction_verification, transaction_verification
+from hathor.verification.block_verifier import BlockVerifier
+from hathor.verification.merge_mined_block_verifier import MergeMinedBlockVerifier
+from hathor.verification.token_creation_transaction_verifier import TokenCreationTransactionVerifier
+from hathor.verification.transaction_verifier import TransactionVerifier
+
+
+class VertexVerifiers(NamedTuple):
+    block: BlockVerifier
+    merge_mined_block: MergeMinedBlockVerifier
+    tx: TransactionVerifier
+    token_creation_tx: TokenCreationTransactionVerifier
+
+    @classmethod
+    def create(cls, *, settings: HathorSettings) -> 'VertexVerifiers':
```

Comment:
Maybe rename to `create_defaults()`.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/796/commits/f35ffcea3b95d6a9ce173a6e5ad18f851aeee033

Reply:
Thanks!